ui/tests/acceptance/jobs-list-test.js

```diff
@@ -356,6 +356,42 @@ module('Acceptance | jobs list', function(hooks) {
     assert.equal(currentURL(), `/csi/volumes?namespace=${namespace.id}`);
   });
 
+  test('when the user has a client token that has a namespace with a policy to run a job', async function(assert) {
```

Comment:
Pretty blocked on how to make a test for this. It doesn't appear that we're easily set-up to write this test since we need to apply the `namespace` to the job but this is a `task-group` test.